lib/plugins/aws/provider/awsProvider.js

```diff
@@ -233,6 +233,10 @@ class AwsProvider {
             UpdateReplacePolicy: { type: 'string' },
             Condition: { type: 'string' },
           },
+          awsLogGroupName: {
+            type: 'string',
+            pattern: '^[a-zA-Z0-9-_.]+$',
```

Comment:
hmmm.. this still doesn't follow pattern at https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-subscriptionfilter.html#cfn-cwl-subscriptionfilter-loggroupname